helm/data_template.go

```diff
@@ -375,7 +375,9 @@ func dataTemplateRead(ctx context.Context, d *schema.ResourceData, meta interfac
 		showOnlyAttrValue := showOnlyAttr.([]interface{})
 
 		for _, showFile := range showOnlyAttrValue {
-			showFiles = append(showFiles, showFile.(string))
+			if s, ok := showFile.(string); ok {
```

Comment:
How about also adding a length check to exclude empty strings?
```suggestion
			if s, ok := showFile.(string); ok && len(showFile) > 0 {
```

Reply:
I see what you mean, we would not only check whether there exists the showFile but we would also exclude any strings that are empty